app/build.gradle

```diff
@@ -36,6 +38,7 @@ android {
         versionName version
         buildConfigField "String", "VERSION", "\"$version\""
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
```

Comment:
Why is multidex necessary? minify is enabled

Reply:
The desugaring process generates a secondary DEX file with the new APIs.